pipx/commands.py

```diff
@@ -327,6 +335,69 @@ def install(
         venv.remove_venv()
         raise
 
+    # if all is well, write out pipxrc file
+    package_or_url_pipxrc = _abs_path_if_local(package_or_url, venv, pip_args)
+    pipxrc = Pipxrc(venv_dir, read=False)
+    pipxrc.set_package_or_url(package_or_url_pipxrc)
+    pipxrc.set_install_options(pip_args, venv_args, include_dependencies)
+    pipxrc.set_venv_metadata(venv.get_venv_metadata_for_package(package))
+    pipxrc.write()
+
+
+def _abs_path_if_local(package_or_url: str, venv: Venv, pip_args: List[str]) -> str:
```

Comment:
let's move this file and the entire pipxrc to another module; no need to be inside commands directly as feels a separate concept than command; additionally rc files doesn't seem a thing AFAIK in python; can we instead make it ``PipxIni`` or ``PipxConf``?

Reply:
Yeah I was already thinking about moving it into pipxrc module.  Just a little weird that it means the setter needs pip_args to work (as opposed to just the package_or_url), but maybe that's not too strange.

As far as the naming goes, I was just starting from what @cs01 had in mind in the first place.  I'm pretty agnostic on that.

Reply:
Also, if we move this into pipxrc, then I definitely need a setter function for package_or_url, because it will need to also accept pip_args as an argument.

Reply:
As far as the name `pipxrc` goes, I think it should be chosen carefully. I still think pipxrc is a decent name but we could probably do better. This name will be used across *everyone's* installation and will be a pain to change down the road.

I personally would not prefer ini (haven't seen an ini file for decades since I was on windows).

Other options (feel free to vote on these or add more). 

```
pipx.json
pipx_metadata.json
pipx_state.json
pipx_package_data.json
```

Reply:
> let's move this function to the pixrc module, or a new one

Agreed. The pipxrc module, or the util.py module might be good.

Reply:
My preference would be the simplest one: `pipx.json`.

Reply:
Ok, moved into pipxrc.  I tried to move it into util.py, but ran into circular reference problems (`abs_path_if_local` needs the `Venv` type, but Venv.py imports util.py).